### ISSUE TITLE
Generator putting files in wrong directory

### ISSUE THREAD

**Issue by sharpmachine** [yeoman/generator-angular#1077]
So I create a project directory within my Sites directory.  Then I cd into my project directory, run yo angular, and the generator puts all the files in my Sites directory.  WTF!

**Comment by eddiemonge** [comment]
Does your `Sites` directory have a `yo-rc.json` file?

**Comment by eddiemonge** [comment]
closing due to inactivity. if this doesn't fix the problem, let me know and we can reopen this

**Comment by vladzzag** [comment]
If directory has ```.yo-rc.json``` file then generator puts all files in the right directory! :)

**Comment by sharpmachine** [comment]
Ahh. So I accidentally created a project outside of my project directory (I
forgot to cd into it before running the yo angular command) so it out the
.yo-rc.json in the Sites directory. I deleted all the files, except the
hidden ones like all the dotfiles. So when I tired to generate the project
in the project directory it kept putting it in the sites directory because
thats where the .yo-rc file was. Once I moved it into my project directory
all was good.
